config/build_simulation.py

```diff
@@ -12,6 +12,16 @@
 from simply.power_network import create_power_network_from_config
 from simply.config import Config
 
+"""
```

Comment:
The doc update in build_simulation.py does not actually fit to the feature addressed in the PR/branch.